plugins/ag-grid/src/js/src/AgGridDhTheme.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  CheckboxStyleParams,
+  InputStyleParams,
+  TabStyleParams,
+} from '@ag-grid-community/theming';
+import { type CoreParams } from '@ag-grid-community/theming/dist/types/src/styles/core/core-css';
```

Comment:
Can't seem to find where to import this type from; it doesn't seem to be exposed anywhere in the current version `v32.3.4`. It is exposed in the latest version though https://app.unpkg.com/ag-grid-community@33.2.4/files/dist/types/src/main.d.ts#L323. 

Is there a particular reason we're using `v32` or should we consider upgrading: https://www.ag-grid.com/javascript-data-grid/upgrading-to-ag-grid-33/

Reply:
Looks like v33 has quite a few breaking changes, including package structure... v33 wasn't available when I made the initial draft. Stick with v32 for now.
I think you can resolve this issue by just having the `type` before the destructuring so it knows the whole import is just for types, e.g.
```suggestion
import type { CoreParams } from '@ag-grid-community/theming/dist/types/src/styles/core/core-css';
```